.github/workflows/build_test.yml

```diff
@@ -16,11 +16,17 @@ jobs:
         runs-on: ${{ matrix.os }}
         steps:
             - uses: actions/checkout@v4
+
             - name: Setup Node
               uses: actions/setup-node@v4
               with:
                   node-version: "18"
                   cache: "npm"
+
+            - name: Update Homebrew Caches
+              if: matrix.os == 'macos-latest'
+              run: brew update
+
             - name: Install Neovim
               uses: rhysd/action-setup-vim@v1
```

Comment:
With this action, homebrew shouldn't be needed.

Why are we using homebrew? It is very slow in CI, it can add minutes.

Reply:
> With this action, homebrew shouldn't be needed.

This action _is_ using homebrew, this is not a new addition. 

> Why are we using homebrew? It is very slow in CI, it can add minutes.

Debatably, we don't need to and could just fetch from the neovim repository directly.

Reply:
> we don't need to and could just fetch from the neovim repository directly.

I thought action-setup-vim was doing that (years ago, that is how it worked...)

https://github.com/neovim/node-client/ and https://github.com/neovim/pynvim fetch from https://github.com/neovim/neovim/releases directly, and if it saves minutes in CI that's probably worth doing.

Reply:
Picking a couple of random examples, it seems that on non-linux platforms, the installation takes ~10s, and this `update` step takes ~15s. Definitely not quick, but not a huge contributor towards our ~8-13 minute runtime.